presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -592,16 +593,17 @@ public HiveConfig setFileStatusCacheTables(String fileStatusCacheTables)
         return this;
     }
 
-    public boolean isHiveViewsEnabled()
+    public boolean isTranslateHiveViews()
     {
-        return hiveViewsEnabled;
+        return translateHiveViews;
     }
 
-    @Config("hive.views-execution.enabled")
-    @ConfigDescription("Allow execution of Hive views")
-    public HiveConfig setHiveViewsEnabled(boolean hiveViewsEnabled)
+    @LegacyConfig("hive.views-execution.enabled")
+    @Config("hive.translate-hive-views")
+    @ConfigDescription("Allow translation of Hive views into Presto views")
```

Comment:
One more thing -- should this (description / toggle name) say it's experimental?
We don't have proper translation layer yet.

Reply:
Good idea. Added.